arkane/statmech.py

```diff
@@ -1618,6 +1622,17 @@ def readScan(self):
                     Es.append(Es[j])
                     xyzs.append(xyzs[j])
 
+            q = len(phis)
```

Comment:
The above lines have the same nested loops. Is it possible to merge these two blocks?

Reply:
duplicate loop fixed

Reply:
Can you double check the fix for the duplicate loops?
I still see two instances of
```python
            for i in xrange(N):  # add the negative values to improve fit near 0.0
                for j in xrange(q):
```
at the same level